libHttpClient.vs2019.sln

```diff
@@ -1,518 +1,358 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28922.388
+# Visual Studio Version 17
```

Comment:
Shouldn't this remain 16?

Reply:
Ya it should. I think I open this in 2022 and it automatically changed it.